core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -345,13 +346,13 @@ describe('Creator dashboard controller', () => {
         expect(ctrl.relativeChangeInTotalPlays).toBe(5);
       });
 
-      it('should change active tab name', function() {
+      it('should change active tab name when clicking on tab', function() {
```

Comment:
Again, "it" is the function being tested. It doesn't click on a tab. So perhaps: "...when the user clicks on a tab".

Note: in cases where you can be more specific, avoid the term "user" and use "learner"/"creator"/"contributor". In this case it is probably "creator".

Reply:
Done.